src/proto/grpc/testing/simple_messages.proto

```diff
@@ -0,0 +1,26 @@
+
+// Copyright 2018 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package grpc.testing;
+
+message SimpleRequest {
+  string message = 1;
```

Comment:
Right and we have a grpc.testing.Empty already :)

Reply:
Let's not encourage re-use of messages across different RPCs. Our best practices says that each RPC should have its own message.

Reply:
I think we talked about this before, but the reason why I am using a constant metadata value here rather than putting the expected value in the message is because I don't want to communicate the expected value through the same mechanism that I'm testing. If there's something wrong with the mechanism, then we could end up with a false positive test result.